Dockerfile-node

```diff
@@ -2,7 +2,7 @@ FROM ubuntu:22.04
 LABEL description="This is the 2nd stage: a very small image where we copy the Substrate binary."
 
 # Install netcat for websocket healthcheck
-RUN apt update && apt install -yq jq netcat
+RUN apt update && apt install -yq jq netcat ca-certificates
```

Comment:
Luckily, we test docker in CI too. After the upgrade, the node was complaining about missing root CAs in the docker.